crates/serde/src/other/arbitrary_.rs

```diff
@@ -31,6 +32,7 @@ impl ArbitraryValue {
         match self {
             Self::Null => serde_json::Value::Null,
             Self::Bool(b) => serde_json::Value::Bool(b),
+            Self::Float(n) => serde_json::Value::String(n.to_string()),
```

Comment:
we should be able to convert this into Value::Number via `Number::from_f64`

Reply:
WDYT?
`Self::Float(n) => serde_json::Number::from_f64(n).into()`

Reply:
that should work, we can treat the None case as 0.0 that should be okay given this is only used for testing